snippets/transpose.md

```diff
@@ -0,0 +1,10 @@
+## Make a transpose matrix
```

Comment:
Please follow the instructions in CONTRIBUTING.md.

Reply:
I'm so sorry, I thought I read it but I missed it.

I would appreciate you checking my changes.